include/wx/string.h

```diff
@@ -1556,13 +1556,13 @@ class WXDLLIMPEXP_BASE wxString
     static wxString FromAscii(const char *ascii, size_t len);
     static wxString FromAscii(const char *ascii);
     static wxString FromAscii(char ascii);
-    const wxScopedCharBuffer ToAscii() const;
+    const wxScopedCharBuffer ToAscii(const char replaceWith='_') const;
```

Comment:
It doesn't make sense to use top-level `const` for the parameter and we don't do it anywhere else (notably in `FromAscii()` just above!), could you please remove it, here and in the documentation?